srcpkgs/nody-greeter/template

```diff
@@ -0,0 +1,36 @@
+# Template file for 'nody-greeter'
+pkgname=nody-greeter
+version=1.3.1
+revision=1
+archs="i686 x86_64"
+build_style=meta
+bootstrap=true
+conf_files="/etc/lightdm/web-greeter.yml /etc/lightdm/Xgreeter /etc/xdg/lightdm/lightdm.conf.d/90-greeter-wrapper.conf"
+hostmakedepends="base-devel nodejs python3"
+makedepends="base-devel libxcb-devel libX11-devel liblightdm-gobject gobject-introspection cairo"
+depends="cairo libX11 libxcb lightdm liblightdm-gobject"
+short_desc="LightDM greeter made with node.js"
+maintainer="JezerM <amyuki4@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/JezerM/nody-greeter"
+distfiles="https://github.com/JezerM/nody-greeter/archive/${version}.tar.gz>${pkgname}-${version}.tar.gz"
+checksum="862a43d0f6a8de5ab071b8e961b811c15d02ed6b7bc0d3adee281d6ae9bbfedf"
+mutable_files="/usr/share/web-greeter/*"
```

Comment:
Does it really alter files in usr/share after they are installed?

Reply:
The user could modify the files inside `/usr/share/web-greeter`, either adding, modifying or deleting themes.